code/game/objects/items/cigs_lighters.dm

```diff
@@ -20,16 +20,18 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	desc = "A simple match stick, used for lighting fine smokables."
 	icon = 'icons/obj/cigarettes.dmi'
 	icon_state = "match_unlit"
+	/// Whether this match has been lit.
 	var/lit = FALSE
+	/// Whether this match has burnt out.
 	var/burnt = FALSE
 	/// How long the match lasts in seconds
-	var/smoketime = 10
+	var/smoketime = 10 SECONDS
 	w_class = WEIGHT_CLASS_TINY
 	heat = 1000
 	grind_results = list(/datum/reagent/phosphorus = 2)
 
 /obj/item/match/process(delta_time)
-	smoketime -= delta_time
+	smoketime -= delta_time SECONDS
```

Comment:
`SECONDS` should really only be for constants, change this to `delta_time * 10` (or `delta_time * (1 SECONDS))`).

Reply:
Done.